spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -6,7 +6,9 @@
   context 'version determination' do
     it 'uses a production version as a default' do
       create(:version, :production)
-      create(:institution, :contains_harv)
+      create(:institution, :contains_harv, approved: true)
+      # adding a non approved institutions row
+      create(:institution, :contains_harv, approved: false)
```

Comment:
You can remove lines 10 and 11.  This spec is just testing that `it 'uses a production version as a default'` and having a non-approved row does not affect the test either way.